ragna/deploy/_ui/main_page.py

```diff
@@ -38,6 +42,9 @@ def __init__(self, api_wrapper, template):
 
     async def refresh_data(self):
         self.chats = await self.api_wrapper.get_chats()
+        self.components = await self.api_wrapper.get_components()
```

Comment:
On UI startup, we retrieve all the information that is needed to create a chat from the UI.

Note that `refresh_data` is also called after a chat has been created, and so these fields should always be accurate.